sunkit_spex/extern/rhessi.py

```diff
@@ -93,7 +95,7 @@ def load_prepare_spectrum_srm(self, spectrum_fn, srm_fn):
             self.start_data_time, self.end_data_time
 
         # Assume unattenuated SRM until spectroscopy interval is specified
-        UNATTENUATED_STATE = 0
+        UNATTENUATED_STATE = 1
```

Comment:
should prob default to first state?

Reply:
the first state is state 0, 1 means thin attenuator inserted

Reply:
Sure but in the other RHESSI example SRM its 1 so needs to work with either.